docs/configuration/config-file-reference.template

```diff
@@ -24,6 +24,7 @@ To specify which configuration file to load, pass the `-config.file` flag at the
 * `<string>`: a regular string
 * `<url>`: an URL
 * `<prefix>`: a CLI flag prefix based on the context (look at the parent configuration block to see which CLI flags prefix should be used)
+* `<relabel_config>`: a [prometheus relabeling configuration](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config).
```

Comment:
```suggestion
* `<relabel_config>`: a [Prometheus relabeling configuration](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config).
```

Reply:
Sorry, I've committed this thinking that I will fix DCO right after, but obviously I cannot push to your branch. 🤦

Reply:
(Well, except via GitHub UI apparently)

Reply:
Yes you can

Reply:
You're correct. I got confused there. Pushed DCO fix, when CI passes, I'll merge.